### PR TITLE
Modal Size static property

### PR DESCRIPTION
**Pull request by wakooka**

### CODE REVIEW

**Comment by Tuxosaurus** [comment]
Do we keep `forwardedRef` in the defaultProps?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Unreleased
 
-[..]
+- **[BREAKING CHANGE]** Removed forwarded props from `Modal`
```
Comment (mcellier) [inline CHANGELOG.md]: on `Modal`?

**Comment by wakooka** [comment]
@Tuxosaurus Yes used by `ConfirmationModal` and `SuccessModal` which pass down `forwardedRef` prop to `Modal`

File: CHANGELOG.md (hunk 1 above)
Comment (wakooka) [reply CHANGELOG.md]: done

**Pull request merged**